Content.Shared/_RMC14/OrbitalCannon/OrbitalCannonSystem.cs

```diff
@@ -380,6 +427,24 @@ private void CannonStatusChanged(Entity<OrbitalCannonComponent> cannon)
         RaiseLocalEvent(cannon, ref ev, true);
     }
 
+    private bool TileHasIndestructibleWalls(EntityCoordinates coordinates)
+    {
+        var anchoredEntities = _rmcMap.GetAnchoredEntitiesEnumerator(coordinates);
+
+        while (anchoredEntities.MoveNext(out var entity))
+        {
+            // This part is shitty because there might be a wall that just... doesn't exactly go with this logic. Hope it works.
+            if (HasComp<TagComponent>(entity) &&
+                _tags.HasTag(entity, "Wall") &&
+                !EntityManager.HasComponent(entity, EntityManager.ComponentFactory.GetRegistration("Destructible")))
```

Comment:
If this is to check whether or not a wall can be destructible since destructible isn't in shared, do a check for damageable instead
All indestructible walls should not be damageable either